jd-geek-source/src/app/components/icons/angle-right-icon/angle-right-icon.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from "@angular/core";
+import { faAngleRight } from "@fortawesome/free-solid-svg-icons";
+
+@Component({
+  selector: "app-angle-right-icon",
+  templateUrl: "./angle-right-icon.component.html",
```

Comment:
for something this small, it is probably easier to just use template string instead of a separate html file...

instead of templateUrl: "...",
use template: `<fa-icon [icon]="faCartPlus"></fa-icon>`,